Doc/whatsnew/3.12.rst

```diff
@@ -413,6 +413,10 @@ Deprecated
   is no current event loop set and it decides to create one.
   (Contributed by Serhiy Storchaka and Guido van Rossum in :gh:`100160`.)
 
+* In accordance with PEP 699, the ``ma_version_tag`` field in :c:type:`PyDictObject`
+  is deprecated for extension modules. Accessing this field will generate a compiler
+  warning at compile time. This field will be removed in Python 3.13.
```

Comment:
Ah wait sorry, I forgot we're already in 3.12. So this should be changed to say removed in 3.14. Please update the news item as well.

Reply:
yes i'll do that